src/execution/physical_plan_generator.cpp

```diff
@@ -5,6 +5,9 @@
 #include "duckdb/main/client_context.hpp"
 #include "duckdb/planner/expression/bound_function_expression.hpp"
 
+//Why is this needed here for compile?
+#include "duckdb/planner/operator/logical_show.hpp"
```

Comment:
Rather than including the file here you should add the forward declaration `class LogicalShow;` to `logical_tokens.hpp`; that should fix the compilation.